packages/internal-workflow-common/src/activity-options.ts

```diff
@@ -87,6 +80,71 @@ Note that the Temporal Server doesn't detect Worker process failures directly. I
   cancellationType?: coresdk.workflow_commands.ActivityCancellationType;
 }
 
+/**
+ * Options for local activity invocation
+ */
+export interface LocalActivityOptions {
+  /**
+   * RetryPolicy that define how activity is retried in case of failure. If this is not set, then the SDK-defined default activity retry policy will be used.
```

Comment:
```suggestion
   * RetryPolicy that defines how an activity is retried in case of failure. If this is not set, then the SDK-defined default activity retry policy will be used.
```